config/config.default.json

```diff
@@ -25,7 +25,9 @@
         "DBInstanceEngine": "mysql",
         "DBInstanceClass": "db.r4.large",
         "DBUserName": "mmuser",
-        "DBPassword": "mostest"
+        "DBPassword": "mostest",
+        "MattermostDownloadURL": "https://releases.mattermost.com/5.19.1/mattermost-5.19.1-linux-amd64.tar.gz",
+        "MattermostLicenseFile": "path/to/license"
```

Comment:
Thoughts on leaving this empty? I think we can document it properly somewhere else.

Reply:
Done